rateLimiter/src/main/java/com/flipkart/varadhi/qos/entity/ClientLoadInfo.java

```diff
@@ -0,0 +1,30 @@
+package com.flipkart.varadhi.qos.entity;
+
+
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class ClientLoadInfo {
+    // current millis
+    private String clientId;
+    private long from;
+    private long to;
+    private List<TrafficData> topicUsageList; // topic to incoming traffic map
```

Comment:
If we are sending topic names every second. What is the throughout requirements from each producer to controller?